docs/pages/preview/upcoming-releases.mdx

```diff
@@ -23,6 +25,44 @@ The team focuses on cleanup of customer-facing issues and documentation.
 You can find the full list of fixes and features in the
 [Github milestone](https://github.com/gravitational/teleport/milestone/55).
 
+## Teleport 7.0 "Stockholm"
+
+The Core team adds support for hardware security modules - HSM for managing certificate authorities.
+The team adds automatic fleet upgrades, improves access requests handlers,
+improves first-time user experience of `teleport` and `tsh`.
```

Comment:
I'd remove fleet upgrades since it looks like we'll only have time to do foundational work and security upgrades.

Some things that are missing: user locking and additional FedRAMP controls.